src/utils/CtypeUtils/CtypeUtils.ts

```diff
@@ -64,23 +95,25 @@ export const getLocalized = (o: any, lang?: string): string => {
  * @returns The CTYPE input model.
  */
 
-export const getCTypeInputModel = (ctype: sdk.CType): ICTypeInput => {
+export const getCTypeInputModel = (ctype: CTypeWithMetadata): ICTypeInput => {
   // create clone
-  const result = JSON.parse(JSON.stringify(ctype.schema))
+  const result = JSON.parse(JSON.stringify(ctype.cType.schema))
   result.$schema = sdk.CTypeInputModel.$id
-  result.title = getLocalized(ctype.metadata.title)
-  result.description = getLocalized(ctype.metadata.description)
+  result.title = getLocalized(ctype.metaData.metadata.title)
+  result.description = getLocalized(ctype.metaData.metadata.description)
   result.required = []
   result.properties = []
 
-  Object.entries(ctype.schema.properties as object).forEach(([key, value]) => {
-    result.properties.push({
-      title: getLocalized(ctype.metadata.properties[key].title),
-      $id: key,
-      type: value.type,
-    })
-    result.required.push(key)
-  })
+  Object.entries(ctype.cType.schema.properties as object).forEach(
```

Comment:
Should we remove the properties `as object`?

Reply:
not part of this ticket